tests/server/login.test.js

```diff
@@ -0,0 +1,88 @@
+const myApp = require("../../server/index");
+const supertest = require("supertest");
+const request = supertest(myApp);
+const sqlite3 = require("sqlite3");
+const Crypto = require("crypto");
+
+// mock the database module
+jest.mock("../../server/db", () => ({
+  get: jest.fn(),
+}));
+
+const db = require("../../server/db");
```

Comment:
I could be wrong, but I _think_ `require` statements are automatically hoisted to the start of the file, so you could likely move this line to where the other requires are. If anything breaks after doing that, just move it back, and then we know :)

Reply:
Jest most likely has some internal hoisting going on to make sure the `jest.mock` is called first anyway